lib/shopify_cli/git.rb

```diff
@@ -63,18 +63,16 @@ def sha(dir: Dir.pwd, ctx: Context.new)
       #
       #   ShopifyCLI::Git.clone('git@github.com:shopify/test.git', 'test-app')
       #
-      def clone(repository, dest, ctx: Context.new)
+      def clone(repository, dest, ctx: Context.new, &block)
         if Dir.exist?(dest) && !Dir.empty?(dest)
           ctx.abort(ctx.message("core.git.error.directory_exists"))
         else
           repo, branch = repository.split("#")
-          success_message = ctx.message("core.git.cloned", dest)
-          CLI::UI::Frame.open(ctx.message("core.git.cloning", repo, dest), success_text: success_message) do
-            if branch
-              clone_progress("clone", "--single-branch", "--branch", branch, repo, dest, ctx: ctx)
-            else
-              clone_progress("clone", "--single-branch", repo, dest, ctx: ctx)
-            end
+
+          if branch
```

Comment:
I believe we could remove this condition by having a method that builds the `git_command` based on `repository`.